docs/en/sql-reference/table-functions/view.md

```diff
@@ -0,0 +1,61 @@
+## View {#view}
+
+Turns an subquery into a table object.
```

Comment:
This function works similarly to the [view](https://clickhouse.tech/docs/en/engines/table-engines/special/view/) table engine. There is no much information, but it can help to make the description more informative.